fe/fe-core/src/main/java/org/apache/doris/catalog/Table.java

```diff
@@ -661,4 +657,9 @@ public List<Long> getChunkSizes() {
     public long fetchRowCount() {
         return 0;
     }
+
+    @Override
+    public List<Pair<String, String>> getColumnIndexPairList(Set<String> columns) {
+        return null;
```

Comment:
return null is very danger. Api such as containsAll could not handle null. return empty list or remove the default impl?